core/src/test/scala/org/apache/spark/sql/delta/DeltaSourceSuite.scala

```diff
@@ -337,6 +337,37 @@ class DeltaSourceSuite extends DeltaSourceSuiteBase with DeltaSQLCommandTest {
     }
   }
 
+  test("maxFilesPerTrigger: Trigger.AvailableNow") {
```

Comment:
whats this got to do with `maxFilesPerTrigger`?

Reply:
actually.. never mind. I was confused with the semantics of AvailableNow

Reply:
actually would be good to spell out the semantics in the test name, that availablenow respects the maxFilesPerTrigger

Reply:
Updated the name